appveyor.yml

```diff
@@ -0,0 +1,52 @@
+environment:
+  global:
+    PROJECT_NAME: OpenAOE
+  matrix:
+    - TARGET: x86_64-pc-windows-gnu
+      BITS: 64
+      CHANNEL: nightly
+      SDL_TARGET: x86_64-w64-mingw32\lib
+      SDL_COPY_PATTERN: libSDL2*
+
+install:
+  - ps: Start-FileDownload "https://static.rust-lang.org/dist/rust-nightly-${env:TARGET}.exe"
+  - rust-nightly-%TARGET%.exe /VERYSILENT /NORESTART /DIR="C:\Program Files (x86)\Rust"
+  - SET PATH=%PATH%;C:\Program Files (x86)\Rust\bin
+  - SET PATH=%PATH%;C:\msys64\mingw64\bin;C:\msys64\usr\bin
+  - SET PATH=%PATH%;SDL2-2.0.4\${env:SDL_TARGET}\bin
```

Comment:
This PATH cannot be relative but I do not know the absolute path of the current working directory.